### ISSUE TITLE
Mass file download for editorss

### ISSUE THREAD

**Issue by elfeto** [MIT-LCP/physionet-build#499]
There should be a way to allow editors download the files, either create a zip and allow download, or a scp/rsync

**Comment by bemoody** [comment]
Yup.  Obviously administrators have shell accounts & can transfer the
files with scp, rsync, etc., but we may want to give this access to
external reviewers too (and make it more convenient for admins while
we're at it.)

Two ideas come to mind:

 - SCP, SFTP, and/or rsync-over-ssh using a project-specific username
and public key (requires a mechanism for user to specify the key, a
mechanism for sshd to associate the username and key with a project,
and to restrict access to that project, probably using chroot.)

 - HTTPS using wget -r (requires a mechanism for authentication; could
perhaps be HTTP Basic - which would also work for WFDB - but for
various reasons I think we don't want to permit HTTP Basic for *web
browser* use so it's a little tricky.  And requires setting up a
directory tree that can be wgotten, but that's easy.)

**Comment by alistairewj** [comment]
It's come up a few times now that users have uploaded a single compressed file. It would be nice to have the inverse of this issue as well; the ability to decompress files on the server without shell access.

**Comment by bemoody** [comment]
Yep.  One of several issues blocked on quota support (issue #518).
